src/content/moment.md

```diff
@@ -0,0 +1,9 @@
+# YOU MIGHT NOT NEED MOMENTJS
+
+Moment is a great library, well crafted, battle tested and supported by a really skilled team, but sometimes folks imports it for convenience, to apply only trivial transformations.
```

Comment:
You should add the `size` of `moment` that is the real problem of the library, especially if used with local.